File: Core/Entities/Part.cs

using System.Collections.Generic;


namespace Core.Entities
{
    public class Part : BaseEntity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public ICollection<PartPhoto> Photos { get; set; }
        public PartBrand PartBrand { get; set; }
        public int PartBrandId { get; set; }
        public PartType PartType { get; set; }
        public int PartTypeId { get; set; }
        public string Manufacturer { get; set; }
        public string Specs { get; set; }
        public string LocalId { get; set; }
        public string PartNumber { get; set; }

    }
}

File: client/src/app/shared/models/part.ts
import { IPartPhoto } from './PartPhoto'

export interface IPart {
    id: number;
    name: string;
    description: string;
    price: number;
    photoUrl: string;
    partType: string;
    partBrand: string;
    localId: string;
    specs: string;
    photos: IPartPhoto[];
    partNumber: string;
    manufacturer: string;
}